timeseries/src/autogluon/timeseries/models/autogluon_tabular/mlforecast.py

```diff
@@ -398,9 +397,14 @@ def _add_gaussian_quantiles(self, predictions: pd.DataFrame, repeated_item_ids:
         normal_scale_per_timestep = pd.Series(np.tile(sqrt_h, num_items), index=repeated_item_ids)
 
         residuals_std_per_timestep = self._residuals_std_per_item.reindex(repeated_item_ids)
-        # Use avg_residuals_std in case unseen item received for prediction
-        if residuals_std_per_timestep.isna().any():
-            residuals_std_per_timestep = residuals_std_per_timestep.fillna(value=self._avg_residuals_std)
```

Comment:
After the refactor in #4460, we used the **scaled** `_avg_residuals_std` across all new items, which sometimes resulted in extreme prediction intervals for items not seen during training. For example, if items have very different scales, using the avg. scale across all items can result in too wide/too narrow intervals.

For example `cif_2016`, `prediction_length=12`, `eval_metric="SQL"`:
- AG v1.1.1
```
              model  score_test  score_val
0  RecursiveTabular    -1.01039   -0.86496
```
- Current `master`
```
              model    score_test  score_val
0  RecursiveTabular -1.140704e+06  -0.862594
```
- This PR
```
              model  score_test  score_val
0  RecursiveTabular   -1.008482  -0.862594
```